static/js/components/MultipleClassificationsForm.jsx

```diff
@@ -0,0 +1,363 @@
+            id={classification.class}
+            aria-labelledby={classification.class}
+            valueLabelDisplay="auto"
+            step={0.25}
+            marks
+            min={0}
+            max={1.0}
+          />
+          {renderSliders(
+            classification.subclasses,
+            depth + 1,
+            [classification.class].concat(path)
+          )}
+        </div>
+      ));
+      return <StyledSlider key={`${classification.class}`} />;
+    });
+
+  // Helper function to loop through array while waiting for
+  // each item to finish an async function
+  // Adapted from: https://codeburst.io/javascript-async-await-with-foreach-b6ba62bbf404
+  const asyncForEach = async (array, callback) => {
+    for (let index = 0; index < array.length; index += 1) {
+      // eslint-disable-next-line no-await-in-loop
+      await callback(array[index], index, array);
+    }
+  };
+
+  const handleSubmit = async () => {
+    setSubmissionRequestInProcess(true);
+    const results = [];
+
+    Object.entries(formState).forEach(
+      // Submit non-zero classifications for each taxonomy
+      ([taxonomy, classifications]) => {
+        const toPost = Object.entries(classifications)
+          // Only submit non-zero classifications that have been
+          // edited (depth > -1)
+          .filter(
+            ([, { depth, probability }]) => probability > 0 && depth > -1
+          );
+        // Post lower depths first (more specific classifications will be added
+        // later, to be the most recent when fetched)
+        toPost.sort((a, b) => a[1].depth - b[1].depth);
+        asyncForEach(toPost, async ([classification, { probability }]) => {
+          const data = {
+            taxonomy_id: taxonomy,
+            obj_id: objId,
+            classification,
+            probability,
+            group_ids: [groupId],
+          };
+          const result = await dispatch(Actions.addClassification(data));
+          results.push(result);
+        });
+
+        // Reset the depths for the posted classifications so that they
+        // are not reposted upon further edits
+        const newFormState = { ...formState };
+        toPost.forEach(([classification, { probability }]) => {
+          newFormState[selectedTaxonomy.id][classification] = {
+            depth: -1,
+            probability,
+          };
+        });
+        setFormState(newFormState);
+      }
+    );
+
+    setSubmissionRequestInProcess(false);
+    if (results.every((result) => result.status === "success")) {
+      dispatch(showNotification("Classifications saved."));
+    }
+  };
+
+  return (
+    <div className={classes.container}>
+      <Typography variant="h6">Edit Classifications</Typography>
+      <FormControl className={classes.taxonomySelect}>
+        <InputLabel id={`taxonomy-select-label-${objId}`}>
+          Select Taxonomy
+        </InputLabel>
+        <Select
+          labelId={`taxonomy-select-label-${objId}`}
```

Comment:
This should prevent the form from adding that annoying padding to <body>:
```suggestion
          labelId={`taxonomy-select-label-${objId}`}
          inputProps={{ MenuProps: { disableScrollLock: true } }}
```